schema/deploy/trigger_functions/commit_project_revision.sql

```diff
@@ -7,6 +7,10 @@ returns trigger as $$
 declare
 begin
 
+  if (new.change_status='committed') and (new.project_id is not null) and (new.change_reason is null) then
+    raise exception 'Cannot commit change with change_reason %', new.change_reason;
```

Comment:
won't new.change_reason always be null in this exception? So the exception raised will print "cannot commit change with change_reason"
```suggestion
    raise exception 'Cannot commit change with null change_reason';
```